src/app/components/cohort-selection/cohort-selection.controller.js

```diff
@@ -108,22 +122,44 @@ angular.module('transmartBaseUi')
                         var elId = '#' + vm.mainContainerId;
                         // Get width of the full gridster grid
                         var _gWidth = angular.element(elId).width();
+                        if(_gWidth <= 0) {
```

Comment:
look here: when the cohort-selection tab is not focused, its width is zero, this causes the available number of columns to be Infinity, thus resulting the frozen window.